ios/app/brave_core_main.mm

```diff
@@ -447,14 +448,15 @@ - (IpfsAPI*)ipfsAPI {
 
 - (void)initializeP3AServiceForChannel:(NSString*)channel
                          weekOfInstall:(NSString*)weekOfInstall {
-#if BUILDFLAG(BRAVE_P3A_ENABLED)
-  _p3a_service = base::MakeRefCounted<brave::BraveP3AService>(
+#if defined(OFFICIAL_BUILD)
```

Comment:
let's not use OFFICIAL_BUILD like this. I think we should only hide the actualy uploading code under this flag

Reply:
as discussed via DM, switched back to using `BRAVE_P3A_ENABLED` and using only to enable/disable service creation